project1/application.py

```diff
@@ -38,5 +38,6 @@ def register():
 
 @app.route("/admin", methods=["GET"])
 def fetch_users():
-    data=db.execute("SELECT * FROM users").fetchall()
+    data = User.query.all()
+    print(data)
```

Comment:
Is there a way to not print when the code is in production.

Reply:
I have just used that print statement for testing purposes. I'll remove that from the code.

Reply:
How about replacing it with a logging module? So that you can turn on the debug mode when needed.

Reply:
Okay, I will make use of it.